Makefile

```diff
@@ -33,7 +30,12 @@ freeze-requirements: virtualenv requirements-dev requirements-app.txt
 	${VIRTUALENV_ROOT}/bin/python -m dmutils.repoutils.freeze_requirements requirements-app.txt
 
 .PHONY: test
-test: test-requirements test-flake8 test-migrations test-unit
+test: test-bootstrap test-requirements test-flake8 test-migrations test-unit
+
+.PHONY: test-bootstrap
+test-bootstrap: virtualenv
+	dropdb -h ${DATABASE_HOST} --if-exists digitalmarketplace_test
+	createdb -h ${DATABASE_HOST} digitalmarketplace_test
```

Comment:
Putting this in as a prerequisite for `test` seems like it's just going to cause trouble with e.g. someone who doesn't use the default user for their db or needs to use a postgres that's on a different port or something. Previously `bootstrap.sh` was never called automatically.